website/static/js/remote/knockout.remote.js

```diff
@@ -59,9 +59,17 @@ function onWsClose() {
 
 function wsActionUpdatePlaylistTrack(newPlaylistTracks) {
   mappedPlaylistTracks = $.map(newPlaylistTracks, function(item) {
```

Comment:
functionnal js precognise to use Array.filter() and Array.map() for this behavior :D

(personnal thinking: i prefer using native method that jQuery methods with useless fallbacks (IE5,6,7) and less optimisation)